tendermint/src/lite_impl.rs

```diff
@@ -0,0 +1,5 @@
+//! Implementation of all light client traits.
+
```

Comment:
Maybe `lite_impl` isn't the best name?

Reply:
Once we move the current `lite` into it's own crate, can we just call this `lite` ?

Reply:
Yes, thought so too.